integrations/Vero/browser.js

```diff
@@ -58,14 +103,30 @@ class Vero {
       logger.error("[Vero]: Event name from track call is missing!!===");
       return;
     }
-    window._veroq.push(["userId", { properties }]);
+    window._veroq.push(["track", event, properties]);
+    const tags = message.context.integerations?.Vero?.tags;
+    const email =
+      get(message, "context.traits.email") ||
+      get(message, "traits.email") ||
+      get(message, "properties.email");
+    const userId = message.userId || message.anonymousId || email;
+    if (isDefinedAndNotNull(tags)) this.addOrRemoveTags(tags, userId);
```

Comment:
For track `tag` support is not required we are supporting only via identify

Reply:
Done